src/NextjsStaticInvalidator.ts

```diff
@@ -0,0 +1,76 @@
+import * as fs from 'fs';
+import * as path from 'path';
```

Comment:
Can we not do something like https://github.com/jetbridge/cdk-nextjs/commit/0e3383ba77645fbb598ea2f053f1cc104e019bf4#diff-30ed025b999f9ebc85cddf6cb620f803d5e26579aa88301435e186eb6e449f6bR109 ?

Reply:
I originally tried adding the static pages in the rewriter logic but it did not work. I don't think the S3 rewriter depends on the CF distribution for the server lambda?
Also, I'm confused on your `createValidation` works w/o needing to tell CloudFormation that the status is SUCCESS/FAILED. 
Without that logic, my deployment would hang forever waiting for status.

Reply:
I have noticed that the S3Rewriter process takes a longer time. I wonder if we add the update request to CloudFormation, it would save some time waiting for the status.

Reply:
I don't think invalidation is as important for static assets because the .js and I think .html files have hashes as part of their names, public files idk about

Reply:
The invalidation for the static pages are important b/c they go through the lambda to retrieve the static pages.
For example, if you have a static page, it will cache it to the browser. When you update that static page, you must invalidate the path in order for the browser to pick up the changes, otherwise the browser will continue to see the old pages until their cache expires.